extensions/schema-compare/src/test/utils.test.ts

```diff
@@ -0,0 +1,105 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as should from 'should';
+import * as azdata from 'azdata';
+import * as mssql from '../../../mssql';
+import {getEndpointName, verifyConnectionAndGetOwnerUri } from '../utils';
+
+// Mock test data
+const mockConnectionProfile: azdata.IConnectionProfile = {
```

Comment:
Can the mock test data be moved to a common file since schemaCompare.test.ts also uses these?

Reply:
Sure, will do.

Reply:
Done.